src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ObsoleteReferences.targets

```diff
@@ -0,0 +1,38 @@
+<!--
+***********************************************************************************************
+Microsoft.NET.ObsoleteReferences.targets
+
+WARNING:  DO NOT MODIFY this file unless you are knowledgeable about MSBuild and have
+          created a backup copy.  Incorrect changes to this file will make it
+          impossible to load or build your projects from the command-line or the IDE.
+
+Copyright (c) .NET Foundation. All rights reserved.
+***********************************************************************************************
+-->
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(NETCoreSdkBundledCliToolsProps)' == ''">
+    <NETCoreSdkBundledCliToolsProps>$(MSBuildThisFileDirectory)..\..\..\Microsoft.NETCoreSdk.BundledCliTools.props</NETCoreSdkBundledCliToolsProps>
+  </PropertyGroup>
+
+  <Import Project="$(NETCoreSdkBundledCliToolsProps)" Condition="Exists('$(NETCoreSdkBundledCliToolsProps)')" />
+
+  <ItemGroup>
+    <_ReferenceToObsoleteDotNetCliTool Include="@(DotNetCliToolReference)" />
+    <DotNetCliToolReference Remove="@(BundledDotNetCliToolReference)" />
+    <_ReferenceToObsoleteDotNetCliTool Remove="@(DotNetCliToolReference)" />
+  </ItemGroup>
+
+  <Target Name="_WarnAboutObsoleteDotNetCliToolReferences"
+          BeforeTargets="CollectPackageReferences"
+          Condition=" '$(SuppressObsoleteDotNetCliToolWarning)' != 'true' ">
+    <Warning Code="DOTNET1018"
```

Comment:
How was this code chosen?

Reply:
Found the highest DOTNET code in https://github.com/dotnet/sdk/blob/master/src/Tasks/Common/Resources/Strings.resx and added one.
https://github.com/dotnet/sdk/blob/master/src/Tasks/Common/Resources/Strings.resx#L195

Reply:
I think those are mostly left over from Nat defensively putting a bunch of messages from project.json era dotnet and nuget in strings.resx before v1 loc deadline just in case we needed them. We don't set Code elsewhere but probably should. I'd follow the current pattern here and we can file a bug to assign codes and use throughout.

Reply:
Ok, opened this issue to follow up https://github.com/dotnet/sdk/issues/2065